microsoft-edge/webview2/concepts/multi-profile-support.md

```diff
@@ -0,0 +1,315 @@
+   * [CoreWebView2ControllerOptions Class](/dotnet/api/microsoft.web.webview2.core.corewebview2controlleroptions)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2Environment.CreateCoreWebView2ControllerOptions Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#createcorewebview2controlleroptions)
+   * [CoreWebView2ControllerOptions Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2controlleroptions)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2Environment10::CreateCoreWebView2ControllerOptions method](/microsoft-edge/webview2/reference/win32/icorewebview2environment10#createcorewebview2controlleroptions)
+   * [ICoreWebView2ControllerOptions interface](/microsoft-edge/webview2/reference/win32/icorewebview2controlleroptions)
+
+---
+
+#### Create a WebView2 control that uses the profile
+
+The `CreateCoreWebView2ControllerWithOptions` method creates a WebView2 control and associates it with the profile you specified.  If the specified profile doesn't exist, a new profile will be created.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+<!-- Ref topic breakout: one webpage per overload; covers all overloads of the method, no need for per-overload links -->
+* [CoreWebView2Environment.CreateCoreWebView2CompositionControllerAsync Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createcorewebview2compositioncontrollerasync)
+* [CoreWebView2Environment.CreateCoreWebView2ControllerAsync Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createcorewebview2controllerasync)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+<!-- Ref topic breakout: one webpage per type; very long webpage covers all methods of the type including overloads.  2nd overload's url adds -1 at end -->
+<!-- preserve sequence per Ref webpage.  for clarity + brevity, list param names, lowercased -->
+* [CoreWebView2Environment.CreateCoreWebView2CompositionControllerAsync(parentWindow) Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#createcorewebview2compositioncontrollerasync)
+* [CoreWebView2Environment.CreateCoreWebView2CompositionControllerAsync(parentWindow, options) Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#createcorewebview2compositioncontrollerasync-1)
+* [CoreWebView2Environment.CreateCoreWebView2ControllerAsync(parentWindow, options) Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#createcorewebview2controllerasync)
+* [CoreWebView2Environment.CreateCoreWebView2ControllerAsync(parentWindow) Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#createcorewebview2controllerasync-1)
```

Comment:
Delete this link?  This overload lacks an options param.  For Win32, we specify methods that include "WithOptions" in the name; presumably the equivalent methods for .NET or WinRT would only be the overloads which have an options param.